### ISSUE TITLE
glow can't render markdown but renders perfectly on GitHub and https://markdownlivepreview.com/

### ISSUE THREAD

**Issue by jeffschwartz** [charmbracelet/glow#618]
**Describe the bug**
glow can't render included markdown but included markdown renders perfectly on GitHub and https://markdownlivepreview.com/

**Setup**
Please complete the following information along with version numbers, if applicable.
 - OS macOS
 - Shell  zsh
 - Terminal wezterm
 - Multiplexer zellij
 - Locale en_US

**To Reproduce**
1. Run glow with included file. It wont render correctly but does render perfectly on GitHub and  https://markdownlivepreview.com/

**Source Code**
[bootstrap.md](https://github.com/user-attachments/files/15890694/bootstrap.md)

**Expected behavior**
To render perfectly like when rendered on GitHub and  https://markdownlivepreview.com/

**Screenshots**

**Additional context**

**Comment by meowgorithm** [comment]
Hi! The code you're referencing ([this](https://github.com/user-attachments/files/15890694/bootstrap.md)) is mostly HTML. Unlike a web browser, Glow will only parse and render Markdown.

If we've got this wrong, please feel free to reopen.

**Comment by jeffschwartz** [comment]
@meowgorithm thank you for your response. The markdown document that I provided is a mix of markdown and HTML. Markdown has always supported HTML and I respectfully can't understand your closing this issue unless you are saying that Glow isn't fully makrdown compliant. Have you tried rendering this document and exploring its result in the browser? As I mentioned in my OP, GitHub has no problem rendering the markdown document that I provided nor do numerous online markdown preview tools.
